flink/Chart.yaml

```diff
@@ -1,9 +1,13 @@
 apiVersion: v1
-appVersion: 1.11.2
+appVersion: "1.0.0"
```

Comment:
appVersion should match version of deployed application [Chart.yaml](https://helm.sh/docs/topics/charts/#the-chartyaml-file). This field is optional. Maybe we don't need to set it.